.github/dependabot.yml

```
@@ -0,0 +1,22 @@
+---
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
```

There is no Python in this project yet, only a Dockerfile.
```suggestion
  - package-ecosystem: "docker"
```